### PR TITLE
Improve description in several documents.

### PR DESCRIPTION
**Pull request by Witoso**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Add evaluation license description. Resolves cksource/ckeditor5-commercial#6802.

Docs: More specific information about UBB. Resolves cksource/ckeditor5-commercial#6762. Resolves cksource/ckeditor5-commercial#6782

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/updating/migration-to-cdn/testing-suite.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-menu-title: Testing suite
+menu-title: Vanilla JS testing suite
```
Comment (godai78) [inline docs/updating/migration-to-cdn/testing-suite.md:2]: That's (not) funny, I already did this change at some point during TP. We are dropping things on the go and don't even realize it.

Comment (Witoso) [reply docs/updating/migration-to-cdn/testing-suite.md:2]: Hmm, this was ok on master, it doesn't show as change rn, as I resolved the conflicts.

File: docs/getting-started/integrations-cdn/angular.md [hunk 2]
```diff
@@ -77,18 +77,15 @@ export class AppComponent {
 			Paragraph,
 			Bold,
 			Italic,
```
Comment (Witoso) [inline docs/getting-started/integrations-cdn/angular.md:79]:
```suggestion
			Italic
```

File: docs/getting-started/integrations-cdn/next-js.md [hunk 3]
```diff
@@ -66,21 +66,18 @@ const CustomEditor = () => {
 		Paragraph,
 		Bold,
 		Italic,
```
Comment (Witoso) [inline docs/getting-started/integrations-cdn/next-js.md:68]:
```suggestion
		Italic
```

File: docs/getting-started/integrations-cdn/react-default-cdn.md [hunk 4]
```diff
@@ -60,21 +60,18 @@ const CKEditorDemo = () => {
 		Paragraph,
 		Bold,
 		Italic,
```
Comment (Witoso) [inline docs/getting-started/integrations-cdn/react-default-cdn.md:62]:
```suggestion
		Italic
```

**Review by godai78 — APPROVED** [review]
Mergeable.

**Pull request merged**